main/lsp/requests/completion.cc

```diff
@@ -353,7 +353,7 @@ void LSPLoop::findSimilarConstantOrIdent(const core::GlobalState &gs, const core
 unique_ptr<ResponseMessage> LSPLoop::handleTextDocumentCompletion(LSPTypechecker &typechecker, const MessageId &id,
                                                                   const CompletionParams &params) const {
     auto response = make_unique<ResponseMessage>("2.0", id, LSPMethod::TextDocumentCompletion);
-    if (!config->opts.lspAutocompleteEnabled && !config->opts.lspAutocompleteMethodsEnabled) {
+    if (!config->opts.lspAutocompleteEnabled) {
```

Comment:
It looks like this will always throw an error since `lspAutocompleteEnabled` will default to `false` but we're telling clients we support this request.

Do we still need `lspAutocompleteEnabled`, or is the check used elsewhere to avoid autocompleting non-methods?

Reply:
Good catch, the only check we need is already listed further down in the file:

https://github.com/sorbet/sorbet/blob/68a97bba5dc02c3c1a681c78eaa5dec694fe222f/main/lsp/requests/completion.cc#L457-L461